content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,246 @@ Calendar
+to listed users to show (or hide) individual calendars.
+
+.. image:: calendar/calendar-attendees.png
+   :align: center
+   :alt: View of Attendees section on Calendar app.
+
+Share Availabilities
+--------------------
+
+On the **Calendar** app main dashboard, click the :guilabel:`Share Availabilities`
+button at the top of the page. Select the available times and dates on the calendar to add them as
+options in the invitation. Once availability has been selected, click the :guilabel:`Open` button
+to navigate to the associated meeting.
+
+.. image:: calendar/calendar-meeting-share-availability.png
+   :align: center
+   :alt: Share availability window on Calendar app.
+
+.. note::
+   Within the :guilabel:`Share Availabilities` feature, selecting date and times is only possible
+   on the *Day*, *Week*, and *Year* calendar views.
+
+On the meeting form, users have several configuration options:
+
+In the :guilabel:`Scheduling` field, select a minimum hour window to ensure appointments are
+confirmed a specified amount of time in advance. For example, select :guilabel:`01:00` to require
+attendees to confirm at least one hour before their appointment time.
+
+In the :guilabel:`Allow Cancelling` field, select a minimum hour window before the appointment, if
+attendees need to make a cancellation.
+
+The :guilabel:`Availability on` field enables attendees to book :guilabel:`Users` or
+:guilabel:`Resources`, such as meeting rooms or tables. After selecting
+:guilabel:`Users` or :guilabel:`Resources`, then type in the desired contact or resource in the
+space below.
+
+The :guilabel:`Front-End Display` field is used to choose :guilabel:`No Picture` or
+:guilabel:`Show Pictures` related to the selected user or resource on the appointment page.
+
+If :guilabel:`Resources` has been selected in the :guilabel:`Availability on` field, users have an
+option to :guilabel:`Manage Capacities`.
+
+Tick the checkbox to limit the maximum amount of people that can use the resource at the same time.
+
+The :guilabel:`Assignment Method` field enables the order in which attendees book their time and
+user/resource:
+
+- :guilabel:`Pick User/Resource then Time`
+- :guilabel:`Select Time then User/Resource`
+
+If :guilabel:`Resources` has been selected in the :guilabel:`Availability On` field, a third option
+is available, :guilabel:`Select Time then auto-assign`.
+
+Schedule
+~~~~~~~~
+
+In the :guilabel:`Schedule` tab of the new meeting page, add or delete time slots. The target date
+and time populate as the first time slots.
+
+To add a new time slot, hit :guilabel:`Add a line`. Click into the new blank space under the
+:guilabel:`From` field, then select and enter the new target start date and time, respectively.
+Repeat under the new blank space under :guilabel:`To` to select and enter the new target end date
+and time.
+
+.. image:: calendar/calendar-new-meeting.png
+   :align: center
+   :alt: New meeting page on Calendar app.
```

Comment:
This image does not show the Schedule tab lines, and does not seem relevant– can remove :)
```suggestion
```